tests/unit/customizations/cloudformation/test_yamlhelper.py

```diff
@@ -182,7 +183,7 @@ def test_unroll_yaml_anchors(self):
             '      Foo: bar\n'
             '      Spam: eggs\n'
         )
-        actual = yaml_dump(template)
+        actual = re.sub('%YAML 1.1\n---\n', '', yaml_dump(template))
```

Comment:
Since our output now intentionally has `%YAML 1.1` at the beginning we should probably add an explicit test to ensure it is there. And if our other tests assertions are not expecting it to be there we should have a utility function that trims it off rather than writing a regex in each test that doesn't want it.

For example you could write a `yaml_dump_without_header()` function that calls `yaml_dump` and trims the header off. Then use that instead of `yaml_dump` in the tests that do not expect the header to be there.